src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/Properties/Resources.resx

```diff
@@ -227,4 +227,7 @@
   <data name="builtin_disabled_extension" xml:space="preserve">
     <value>Disabled</value>
   </data>
+  <data name="builtin_main_list_page_searchbar_placeholder" xml:space="preserve">
+    <value>Type here to search commands...</value>
```

Comment:
@chatasweetie Would it make sense to having dome like "Search for apps, files and commands.." to be jnclusive of all the content that can be found? (This is what Start also uses btw)